### ISSUE TITLE
the "units" parameter

### ISSUE THREAD

**Issue by xi-wan** [akash13singh/lstm_anomaly_thesis#9]
Hi! thank you for sharing this amazing code. I have one problem when I try to implement the code.
when I try to run these lines:
multistep_lstm = MultiStepLSTM(look_back=288, look_ahead=1, 
                                layers={'input': 1, 'hidden1':120, 'output':1},
                                dropout=0.1, loss='mse',
                                learning_rate=0.02)
model = multistep_lstm.build_model()
it mentioned that: __init__() missing 1 required positional argument: 'units'

I see you set the self.layers['hidden1'] as units in the function called "StatefulMultiStepLSTM", but set the self.layers['hidden1'] as output_dim in the function"MultiStepLSTM". 

can you help me with the missing argument problem?
